applications/dashboard/controllers/class.messagecontroller.php

```diff
@@ -201,6 +219,7 @@ protected function setEnabled($messageID, $enabled) {
         } else {
             $this->informMessage(sprintf(t('%s disabled.'), t('Message')));
         }
+        Gdn::cache()->remove('Messages');
```

Comment:
`Gdn::cache()->remove('Messages')` has obvious functionality.

`MessageModel::messages(null)` is ambiguous unless you know what passing a null does.